test/testdata/certs/cert.conf

```diff
@@ -0,0 +1,31 @@
+# =====================================================================
```

Comment:
You have permission to add key and certificate files to this repository for testing purposes.